### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -112,6 +111,12 @@
           "npmPublish": false
         }
       ],
+      [
+        "@semantic-release/npm",
```
Comment (ilfa) [inline package.json:115]: It's just a name and it works with yarn too?

Comment (TheUnderScorer) [reply package.json:115]: Yes 😄. It is used to bump a version in `package.json` and publish it to NPM

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 name: Release
 
 on:
-  # TODO After publishing repository change this to trigger on push to main
-  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - test
```
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply .github/workflows/release.yml:7]: Yes, push to `test` branch will tag the release as pre-release, push to `main` branch will trigger normal release.

**Review by ilfa — APPROVED** [review]

**Pull request merged**